### ISSUE TITLE
GET result url query param wrong

### ISSUE THREAD

**Issue by svensteudter** [chuckwight/chemvantage#3]
According  Learning Tools Interoperability Assignment and Grade Services Version 2.0 OpenAPI Specs
the GET /result ennpoint has as url query param "user_id" see:
http://www.imsglobal.org/spec/lti-ags/v2p0 chapter 2.3.5:

>   2.3.5 Container Request Filters
A 'GET' request to the container endpoint MAY use the following query parameters to filter the response:

'user_id': to filter the results to a single user. The results MUST contain at most 1 result. An empty array MAY be returned if the user does not have any result recorded. 

And also check against https://www.imsglobal.org/spec/lti-ags/v2p0/openapi/#/default/Results.GET 

ChemVanatge is using in 
https://github.com/chuckwight/chemvantage/blob/7bfa065b88f94295ef9b934510cad137801f62cc/src/org/chemvantage/LTIMessage.java#L557
Also verified in our log:
....lti13/context/712680-1575045/lineitems/1585456/results?userId=2

TL;DR
url query param should be "user_id" for GET /result instead of "userId"

**Comment by chuckwight** [comment]
Fixed by accepting multiple versions of userId
